### PR TITLE
NO-JIRA: More snyk ignores

### PR DESCRIPTION
**Pull request by dulek**

### CODE REVIEW

**Comment by mandre** [comment]
Can you please rebase on top of the new master?
/hold

File: .snyk [hunk 1]
```diff
@@ -7,3 +7,8 @@ exclude:
     - "hack/**"
     - "tests/**"
     - "**/*_test.go"
+    - "charts/**"
+    - "examples/**"
+    - "examples/**"
```
Comment (mandre) [inline .snyk:12]: let's make extra sure we skip examples :smile:

**Comment by dulek** [comment]
@mandre ?

**Pull request merged**